model/src/main/proto/exploration.proto

```diff
@@ -273,6 +273,9 @@ message EphemeralState {
 
   // The timestamp at which continue button should start animating.
   int64 continue_button_animation_timestamp = 10;
+
+  // Determines if the continue button animation should be shown or not.
+  bool show_continue_button_animation = 11;
```

Comment:
Nit: swap this & the above's places and indexes (it makes more sense, I think, for the bool to go before the long since it "gates" whether to do the action at all).

Reply:
done